traffic_ops/app/lib/MojoPlugins/Daemonize.pm

```diff
@@ -41,12 +41,6 @@ sub register {
 				# This is the first child
 				$self->inactivity_timeout(0);
 				POSIX::setsid();
-				open( STDIN, "< /dev/null" )
-					|| confess("Can't read /dev/null: $!");
```

Comment:
Cool.
If child STD streams are not detached from the parent streams, this could be the source of "mysterious" problems in "random" worker processes due to child daemons doing IO, down the line.